src/Symfony/Component/Intl/Data/Generator/LocaleDataGenerator.php

```diff
@@ -138,69 +138,29 @@ public function generateData(GeneratorConfig $config)
 
     private function generateLocaleName($locale, $displayLocale)
     {
-        $name = null;
-
-        $lang = \Locale::getPrimaryLanguage($locale);
-        $script = \Locale::getScript($locale);
-        $region = \Locale::getRegion($locale);
-        $variants = \Locale::getAllVariants($locale);
-
         // Currently the only available variant is POSIX, which we don't want
         // to include in the list
-        if (\count($variants) > 0) {
-            return;
-        }
-
-        // Some languages are translated together with their region,
-        // i.e. "en_GB" is translated as "British English"
```

Comment:
not applicable; we get the language name based on `Locale::getPrimaryLanguage`, which is `en` for `en_GB` already.

Thus we translate the correct language name already.